docker/sbtc/emily-sidecar/Dockerfile

```diff
@@ -1,11 +1,17 @@
-FROM python:3.12-slim AS emily-sidecar
+FROM python:3.13-slim AS emily-sidecar
 
-WORKDIR /app
+# Install uv.
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
```

Comment:
we should probably pin uv version (via a sha hash of the image I guess, given how we are installing it)